docs/release.md

```diff
@@ -0,0 +1,26 @@
+# Release Process
+This document describes how to cut a `cri` plugin release.
+
+## Step 1: Update containerd vendor
+Before cutting a release, update the containerd vendor to latest
+version in the corresponding branch, and make sure all tests in
+CI pass https://k8s-testgrid.appspot.com/sig-node-containerd.
+## Step 2: Cut the release
+Draft a new release in https://github.com/containerd/cri/releases.
```

Comment:
Draft and tag a new release... or should the release tag wait for the updating the vendor.conf to the version of containerd that contains the last code commit?

Reply:
We need to release on one side first. I think it should be `cri`. :)

Reply:
Done